scripts/bee.sh

```diff
@@ -278,9 +297,15 @@ echo "Check whether the queen node has been connected to every worker..."
 ELAPSED_TIME=0
 WAITING_TIME=2
 TIMEOUT=$((2*30*WAITING_TIME))
+RESTRICTED_TOKEN=""
 while (( TIMEOUT > ELAPSED_TIME )) ; do
     check_queen_is_running
-    COUNT=$(count_connected_peers)
+    if [ $RESTRICTED=="true" -a -z $RESTRICTED_TOKEN ]; then
```

Comment:
```suggestion
    if [ $RESTRICTED == "true" ] && [ -z "$RESTRICTED_TOKEN" ]; then
```

Reply:
Why you don't want to use `-a`?

Reply:
because it is ugly and also the ShellCheck underlines.